samples/apps/copilot-chat-app/SKWebApi/Program.cs

```diff
@@ -24,28 +25,42 @@ public static void Main(string[] args)
             serverPort = SKWebApiConstants.DefaultServerPort;
         }
 
+        // Set the protocol to use
         string protocol = "https";
+        bool useHttp = builder.Configuration.GetSection("UseHttp").Get<bool>();
+        if (useHttp)
+        {
+            protocol = "http";
+        }
+
         builder.WebHost.UseUrls($"{protocol}://*:{serverPort}");
 
         // Add services to the DI container
         AddServices(builder.Services, builder.Configuration);
 
         var app = builder.Build();
 
+        var logger = app.Services.GetRequiredService<ILogger>();
```

Comment:
@adrianwyatt Notice that by using Get**Required**Service, I don't have to deal with a potentially null logger